ftw/book/browser/resources/ftwbook.css

```diff
@@ -115,6 +115,36 @@ span.keyword, body.mceContentBody span.keyword {
   padding-right: 1px;
 }
 
+/** TINYMCE footnote plugin **/
+.ploneSkin span.mce_tinymce_footnote {
+  background: url("++resource++ftw.book.tinymce.footnote/img/footnote.png") no-repeat center center;
+}
+
+span.footnote-info {
```

Comment:
Might be possible.
I did a quick test and the `:after` didn't show up. I do some display stuff in there so this might not be possible with the after selector. This also conflicts with the hover which is also in the after. Conclusion: I would rather leave it as it is now.